src/components/navbar/avatar.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
```

Comment:
Move this into the index.js file you will create at components/navbar and export it